packages/js/uri-resolver-extensions/src/UriResolverExtensionFileReader.ts

```diff
@@ -32,5 +32,10 @@ export class UriResolverExtensionFileReader implements IFileReader {
       this.resolverExtensionUri,
       combinePaths(this.wrapperUri.path, filePath)
     );
+    if (!result.ok) return result;
+    if (!result.value) {
+      return ResultErr(new Error(`File not found at ${filePath}`));
```

Comment:
Swapping `filePath` for `combinePaths(this.wrapperUri.path, filePath)` would give a more descriptive error message.